src/Fixer/Comment/AlignMultilineCommentFixer.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Comment;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class AlignMultilineCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Multiline comments and docblocks MUST be aligned with comment opening.',
+            [new CodeSample('<?php
+    /*
+     * Multiline comment
+  *
+ Lines not prefixed with asterisk are left untouched
+       *
+   */')]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound([T_COMMENT, T_DOC_COMMENT]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $intendationChars = 0;
+        foreach ($tokens as $index => $token) {
+            $newlinePosition = mb_strrpos($token->getContent(), "\n");
+
+            if ($token->isComment() && '/*' === substr($token->getContent(), 0, 2) && false !== $newlinePosition) {
+                $content = $token->getContent();
+                $content = preg_replace('/^([ \t]+)?\*/m', str_repeat(' ', $intendationChars).'*', $content);
```

Comment:
The tab issue is the reason I suggested to not fix multiline comments that start on the same line as another statement. I don't see a solution for those but using spaces only.

If the fixer only cares about comments that start on their own line, it will be able to:
- fix indentation of comments start;
- align subsequent lines to this start using configured indentation style.

But, like you said, this kind of comment is often used to disable parts of the code. I don't think all users will expect those to be changed by the fixer, and you have no way to detect whether the comment is expected to be aligned.

So IMO, by default the fixer should only care about multiline comments that:
- are `/**` DocBlocks;
- start on their own line.

Maybe fixing regular `/*` comments and/or comments that start on the same line as another statement could be enabled using options that default to `false`.